### PR TITLE
Clarify prepare statements usage

### PR DESCRIPTION
**Pull request by petroav**

### CODE REVIEW

**Comment by petroav** [comment]
@rschlussel can you take a look please

**Comment by petroav** [comment]
@brian-rickman can you take a look please

File: presto-docs/src/main/sphinx/sql/prepare.rst [hunk 1]
```diff
@@ -17,6 +19,9 @@ are saved on a per-session basis with a given name. The statement can include
 parameters in place of literals to be filled in at execution time. Parameters
 are represented by question marks.
 
+Once a statement is prepared using the :ref:`prepare` syntax above, it can be
+executed using the :ref:`execute` statement and deleted from the session using
+the :ref:`deallocate-prepare` statement.
```
Comment (rschlussel-zz) [inline presto-docs/src/main/sphinx/sql/prepare.rst]: Note describe input and describe output too.

**Comment by rschlussel-zz** [comment]
looks good

File: presto-docs/src/main/sphinx/sql/prepare.rst (hunk 1 above)
Comment (brian-rickman) [inline presto-docs/src/main/sphinx/sql/prepare.rst]: These changes look good.

**Comment by petroav** [comment]
Thanks for the review, merged to release branch and will amend doc commit of `prepared-statements` branch.

**Pull request closed without merging**